src/internal.c

```diff
@@ -4571,6 +4586,12 @@ int RsaVerify(WOLFSSL* ssl, byte* in, word32 inSz, byte** out, int sigAlgo,
         return ret;
 #endif
 
+#if defined(HAVE_PK_CALLBACKS) && defined(WOLFSSL_MAXQ108x)
+    if (ssl->options.side == WOLFSSL_CLIENT_END) {
+        maxq10xx_SetRsaPssSignature(in, inSz);
```

Comment:
Why is this required? Can't the `RsaPssVerifyCb` be used? It passes in and inSz.

Reply:
Excellent suggestion. I will investigate.

Reply:
Fixed.